rpa-builder/6.1/modules/ROOT/pages/_partials/toolbox-web-automation-web-session.adoc

```diff
@@ -128,27 +129,39 @@ automatically.
 +
 The Workflow Run ends in an error.
 
-The standard response of the browser and the execution are defined in
-the *General Settings* area.
+Define the standard response of the browser and the execution in the *General Settings* section.
 
-You use the *Initial URL* to specify the initial URL of the web page you
-want to call.
+=== General Settings 
 
+image:toolbox-variable-web-automation-web-session-general-settings.png[50%, 50%, The General Settings section of the Wizard]
```

Comment:
Though it is an alt text, it would be great to add the full stop at the end of the sentence to improve the pronunciation of the speech machine.